apps/changelog/package.json

```diff
@@ -21,15 +21,14 @@
     "@radix-ui/react-icons": "^1.3.0",
     "@radix-ui/react-toolbar": "^1.0.4",
     "@radix-ui/themes": "^2.0.3",
-    "@sentry/nextjs": "^8.8.0",
-    "@spotlightjs/spotlight": "^2.1.1",
```

Comment:
q: Why did you remove spotlight?

Reply:
It was one more thing I needed to get working with the new Next.js version. I will add it back in a follow-up. But good that you bring it up!